src/Index.cmake

```diff
@@ -1,17 +1,19 @@
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.16)
 
-include(CMakeParseArguments) # to support cmake 3.4 and older
```

Comment:
This comment clarified the intention of this include. In recent versions of Cmake, this is not needed anymore.

Reply:
Than the include should be removed; minimum `CMake` version is 3.16